test/e2e/apimachinery/apigroup_preferred_version.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apimachinery
+
+import (
+	"context"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	"github.com/onsi/ginkgo"
+)
+
+var _ = SIGDescribe("apigroup preferred version", func() {
+	f := framework.NewDefaultFramework("apigroup-preferred-version")
+	ginkgo.It("should validate PreferredVersion for each APIGroup", func() {
+
+		// get list of APIGroup endpoints
+		list := &metav1.APIGroupList{}
+		err := f.ClientSet.Discovery().RESTClient().Get().AbsPath("/apis/").Do(context.TODO()).Into(list)
+
```

Comment:
nit: unnecessary blank line for error handling.

Reply:
Fixed.